src/ReachSets/ContinuousPost/GLGM06/post.jl

```diff
@@ -0,0 +1,47 @@
+function post(𝒜::GLGM06,
+              𝑃::InitialValueProblem{<:AbstractContinuousSystem},
+              𝑂::Options)::ReachSolution{Zonotope}
+
+    # ==================================
+    # Initialization and discretization
+    # ==================================
+
+    𝑂 = merge(𝒜.options.defaults, merge(𝑂, 𝒜.options.specified))
```

Comment:
Since this is a pattern that you seem to like, maybe add the method `merge(𝑂::Options, 2L𝑂::TwoLayerOptions)`.

Reply:
👍  equivalent, while we don't have that method: `𝑂 = merge(𝒜.options.defaults, 𝑂, 𝒜.options.specified)`

Reply:
I created #564.